waltid-services/waltid-issuer-api/src/main/kotlin/id/walt/issuer/issuance/OnboardingRequest.kt

```diff
@@ -22,3 +24,15 @@ data class OnboardingRequest(
         method = "jwk"
     )
 )
+
+fun validateOnboardingRequest(request: OnboardingRequest) {
+    require(request.key.backend.isNotEmpty()) {
+        throw WebException(HttpStatusCode.BadRequest, "Key backend must not be empty")
```

Comment:
I would go for the property name. "Missing property X" or "Property x in the request body cannot be empty"